TraderJoe-Bots/Large-Swap/src/network.ts

```diff
@@ -0,0 +1,61 @@
+interface NetworkData {
+  chainId: number;
+  multicall2: string;
```

Comment:
Why `multicall2` is needed? the address is stored inside `multicall2Data` too which seems to be the one used in the bot logic.